### DISCUSSION TITLE
How to request new version of Spectral with updated packages

### DISCUSSION THREAD

**Discussion by dshiledarc1** [stoplightio/spectral#1530] [General]
Hi 

I created issue 63 for the security findings related to yargs v15.4.1 library, what is the process of getting a new version of Spectral published.

To help out also created a pull request - https://github.com/stoplightio/spectral/pull/1529 

Thanks in advance
Devendra

**Comment by dshiledarc1** [comment]
When is the next version will be published to npm repo? I am waiting for the changes merged 10 days ago by @P0lip to appear. 

Thanks
Devendra

**Reply by P0lip** [reply]
Hey, I'll probably release 5.9.1 soon, maybe next week.

**Reply by dshiledarc1** [reply]
Hi @P0lip thank you, please include netmask https://github.com/stoplightio/spectral/issues/1562 issue as well, as it is preventing the usage of this excellent library.

**Reply by dshiledarc1** [reply]
@BTW the build process works fine on my personal laptop, had issues with it on the corporate machine. I can contribute as needed.

**Reply by P0lip** [reply]
Hey, 5.9.1 is out! 
https://www.npmjs.com/package/@stoplight/spectral/v/5.9.1

> @BTW the build process works fine on my personal laptop, had issues with it on the corporate machine. I can contribute as needed.

awesome, glad it works.
It might have been something off with proxy, i.e. you're behind a corporate proxy, etc.
